lib/component.js

```diff
@@ -93,7 +93,7 @@ class Component extends WebComponent {
    * @example
    * {template: state => h('.header', this.child('my-child-widget'))}
    */
-  child(tagName, config={}) {
+  child(tagName, config = {}) {
```

Comment:
```suggestion
  child(tagName, config={}) {
```
This is the style of the whole lib. If you're going to fiddle with this spacing please do it in a single pass everywhere and make it a separate PR, thanks.